src/store/plugins/load-layersconfig-on-lang-change.js

```diff
@@ -62,6 +70,18 @@ const loadLayersAndTopicsConfigAndDispatchToStore = async (store) => {
         if (topicEch) {
             topicEch.backgroundLayers.push(lightBaseMapBackgroundLayer)
             topicEch.defaultBackgroundLayer = lightBaseMapBackgroundLayer
+            // replacing the SWISSIMAGE WMTS layer with the SWISSIMAGE vector layer
+            // same as the other one above, this should be removed ASAP (as soon as our backend
+            // is serving this configuration through the standard layersConfig endpoint)
+            const swissimageLayer = topicEch.backgroundLayers.find(
+                (layer) => layer.geoAdminID === 'ch.swisstopo.swissimage'
+            )
+            if (swissimageLayer) {
+                log.debug('replacing SwissImage background layer with vector style equivalent')
+                swissimageLayer.isBackground = false
+                topicEch.backgroundLayers[topicEch.backgroundLayers.indexOf(swissimageLayer)] =
+                    imageryBackgroundLayer
+            }
```

Comment:
Based on the comment this should change in the future, I would suggest to create a jira ticket for this task (if not already done) and refer it here. This would make the future change easier to understand/find.